Engine/Engine.cs

```diff
@@ -128,6 +128,9 @@ public void Run(AlgorithmNodePacket job, AlgorithmManager manager, string assemb
                     // initialize the alphas handler with the algorithm instance
                     AlgorithmHandlers.Alphas.Initialize(job, algorithm, SystemHandlers.Notify, SystemHandlers.Api);
 
+                    // initialize the object store
+                    AlgorithmHandlers.ObjectStore.Initialize(algorithm.Name, job.UserId, job.ProjectId, job.UserToken, job.Controls);
```

Comment:
It might be nice to include messaging abilities to report issues. It could be passed in here. `SystemHandlers.Notify`

Reply:
Also, are we still OK with errors being _thrown_ and then notifying the user about issues only during persistence since it's outside of their call stack or have the notifications for _all_ object store errors?